Add wp site meta sub-command

spacedmonkey opened the pull request:
Add the wp site meta sub-command, with tests. 

This one is going to be hard to test and might be merge after 5.0.0 is released. 

fixes #158

schlessera commented on entity-command.php:
```diff
@@ -32,9 +32,6 @@
 			WP_CLI::error( 'This is not a multisite installation.' );
 		}
-		if ( \WP_CLI\Utils\wp_version_compare( '5.0', '<' ) ) {
-			WP_CLI::error( "Requires WordPress 5.0 or greater." );
```
You'd still need to error out if the WordPress version is not the correct one, i.e. 5.0.

spacedmonkey commented:
Now that #174 is merged, I have updated the PR, with overrides that use the `site_meta` functions. 

@schlessera Can you take a look at this PR again. 

Quick note, I have rebased my branch, so I could make this change, it has a lot of commit from master in this branch. Sorry about this.

schlessera commented:
Thanks for the PR, @spacedmonkey !
